python/test/unit/tools/test_aot.py

```diff
@@ -119,7 +118,7 @@ def kernel(C, A, B,
 
 
   // free cuda handles
-  unload_kernel();
+  unload_matmul_fp16xfp16_16x16x16();
```

Comment:
The launcher fn name is changed to `args.out_name`, since people could customize the name to meet different data-type combination scenarios.
For instance, given a Triton kernel called `kernel`, when the input datatypes vary from fp16 to fp32, we can set the `args.out_name` from `kernel_fp16` to `kernel_fp32`, in this way, we can use link.py twice and produce two launchers, one is called `kernel_fp16(...)` and the other is `kernel_fp32(...)`, rather than forcing all of them to be `kernel(...)`.